draft-bar-cfrg-spake2plus.md

```diff
@@ -174,13 +176,16 @@ This sample trace is shown below.
 
 ## SPAKE2+ {#spake2plus}
 
-This protocol appears in {{TDH}} and is proven secure in {{UCAnalysis}}. Let w0
-and w1 be two integers derived by hashing the password pw with the identities
-of the two participants, A and B. Specifically,
+Let w0 and w1 be two integers derived by hashing the password pw with the
+identities of the two participants, A and B. Specifically, compute
 w0s || w1s = PBKDF(len(pw) || pw || len(A) || A || len(B) || B),
-and then computing w0 = w0s mod p and w1 = w1s mod p.
+and then w0 = w0s mod p and w1 = w1s mod p.
 If both identities A and B are absent, then w0s || w1s = PBKDF(pw), i.e.,
 the length prefix is omitted as in {{setup}}.
+If both identities A and B are unknown at the time of deriving w0 and w1,
```

Comment:
What if only one of them is unknown? Is only the unknown identity omitted?

Reply:
Yeah, that was the idea. Will clarify.